core/controllers/reader_test.py

```diff
@@ -430,7 +430,7 @@ def setUp(self):
         self.editor = user_services.UserActionsInfo(self.editor_id)
 
         # Load exploration 0.
-        exp_services.load_demo(self.EXP_ID)
+        # exp_services.load_demo(self.EXP_ID)
```

Comment:
Hi @prasanna08 -- why comment this out? Either remove fully or leave it in.

Reply:
Oops. I intended to remove it.